nixos/common.nix

```
@@ -28,6 +29,7 @@ let
     specialArgs = {
       lib = extendedLib;
       osConfig = config;
+      _osClass = _class;
```

```suggestion
      osClass = _class;
```
Seems like the underscore is intended to avoid conflict with normal usages. We're already prefixing `os`, it should be fine, I think?